psychopy/demos/coder/basic/global_event_keys.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+"""
+This demo shows how to use "global event keys" to interact with a running
+experiment. The keys do not have to be checked manually, but
+rather get handled automatically by PsychoPy.
+
+This works when calling win.flip() repeatedly (as in this example)
+and when using core.wait() (not demonstrated here). If using wait(),
+be sure to set the `hogCPUperiod` parameter equal to the entire
+waiting duration.
+
+"""
+
+from __future__ import print_function
+from psychopy import core, event, visual
+
+
+win = visual.Window()
+rect = visual.Rect(win, fillColor='blue', pos=(0, -0.2))
+text = visual.TextStim(
+    win,
+    pos=(0, 0.5),
+    text=('Press\n\n'
+          'B for blue rectangle,\n'
+          'CTRL + R for red rectangle,\n'
+          'Q or ESC to quit.'))
+
+# Add an event key.
+event.globalKeys.add(key='b', func=setattr,
+                     func_args=(rect, 'fillColor', 'blue'),
+                     name='blue rect')
+
+# Add an event key with a "modifier" (CTRL).
+event.globalKeys.add(key='r', modifiers=['ctrl'], func=setattr,
+                     func_args=(rect, 'fillColor', 'red'),
+                     name='red rect')
+
+# Add multiple keys at once.
+[event.globalKeys.add(key, func=core.quit) for key in ('q', 'escape')]
```

Comment:
Probably better to write it as

    for key in ('q', 'escape'):
        event.globalKeys.add(key, func=core.quit)

The list comprehension implies that you want to make a list. But add() returns nothing and the list is not saved.